unblob/processing.py

```diff
@@ -101,16 +101,21 @@ class ExtractionConfig:
     verbose: int = 1
     progress_reporter: Type[ProgressReporter] = NullProgressReporter
 
-    def get_extract_dir_for(self, path: Path) -> Path:
-        """Return extraction dir under root with the name of path."""
+    def intern(self, path: Path) -> Path:
```

Comment:
function name does not say a lot about its purpose. How about `get_path_under_root` ?

Reply:
Changed to `_get_output_path`.